euclidean_gcd/CPP/euclidean_gcd.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+
+int gcd_rec(int a, int b)
+{
```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*